test/tests/components/Dropdown.js

```diff
@@ -0,0 +1,72 @@
+import expect from 'expect';
+import Dropdown, { Props } from '../../../src/dropdown';
+import { newComponent } from '../helpers';
+
+describe('Dropdown', function () {
+
+  describe('locals', () => {
+    const exampleProps = {
+      id: '12345',
+      className: 'fancy-class-name',
+      style: { margin: 10, position: 'relative' },
+      value: { value: 'test', label: 'Test' },
+      onChange: () => {},
+      options: [
+        { value: 'test', label: 'Test' },
+        { value: 'test1', label: 'Test1' },
+        { value: 'test2', label: 'Test2' }
+      ]
+    };
+    const componentDropdown = new Dropdown(exampleProps);
+
+    it('should pass props', ()  => {
+      const locals = componentDropdown.getLocals();
+      expect(locals.id).toBe(exampleProps.id);
+      expect(locals.style).toEqual(exampleProps.style);
+      expect(locals.options).toEqual(exampleProps.options);
+      expect(locals.onChange).toEqual(exampleProps.onChange);
+    });
+
+    it('should compute className', () => {
+      const { className } = componentDropdown.getLocals();
+      expect(className).toInclude('dropdown');
+      expect(className).toInclude('fancy-class-name');
+    });
+
+    it('should compute value', () => {
```

Comment:
`should leave object value untouched` (we're testing that if you pass `value` as an object it remains untouched)